Source/Core/Core/Core.cpp

```diff
@@ -891,58 +891,13 @@ void Callback_NewField()
 
 void UpdateTitle()
```

Comment:
Make sure to update `VideoThrottle()` on line 846 and change the rate at which the title is updated to maybe, once every 5 seconds.

Reply:
Do we actually still need this after removing all of it? Isn't the remainder simply static and can be set _once_ without requiring any updates afterwards? Or, at least, only during defined times (such as a config changed calledback, since we _do_ list the DC/SC setting along with the display backend and HLE/LLE audio)

Reply:
I'm actually in favor of removing all the settings information too, so it's only `{version} + " | " + {game title}`. But that might remove too much information for some people.

Reply:
After some discussion on Discord, we decided to keep this PR simple and only remove the FPS, VPS, and speed %. I will update the code later only to refresh the title if the config values have been updated.

Reply:
I increased the update frequency of the title bar text to 1000ms because it isn't necessary to update it at a higher rate. @OatmealDome and I didn't want to remove this yet because some OSD messages show up in the window title bar and the periodic update is used to restore the original title. Changing this behavior should be done in another PR as it requires some separate discussion.

Reply:
And that PR is #11471 ~

Reply:
Are there any changes you want to make to this PR now that #11471 is merged?

Reply:
@MayImilae Updating the window title every 1000ms doesn't seem very clean now that the PR with the window title OSD has been merged. I will update this PR later to only update the window title when necessary (when the game changes and when the configuration is updated).

Reply:
OK should be much cleaner now.